src/server/pfs/server/api_server.go

```diff
@@ -616,6 +616,11 @@ func (a *apiServer) ListFileV2(req *pfs.ListFileRequest, server pfs.API_ListFile
 	return errors.Errorf("v2 method not implemented")
 }
 
+// GlobFileV2 not implemented
+func (a *apiServer) GlobFileV2(req *pfs.GlobFileRequest, server pfs.API_GlobFileV2Server) error {
+	return errors.Errorf("method not implemented")
```

Comment:
This error message should be consistent with the others. Maybe this should be a constant that is returned by each of the V2 methods.